### PR TITLE
Added attachment previews for images.

### LINKED ISSUE #577: Image attachments should be opened in browser instead of downloaded
In order to view images attached to tickets, you first have to download them. Images should be displayed directly in the browser.

### PR DESCRIPTION
**Pull request by brettp**
Fixes #577.

### CODE REVIEW

**Comment by superalex** [comment]
Hello @brettp,

Your PR looks great!, I've just added two comments that we would like to check before merging

Thank you very much!

File: app/styles/modules/common/attachments.scss [hunk 1]
```diff
@@ -183,3 +183,7 @@
         color: $gray-light;
     }
 }
+
+.attachment-preview img {
+    max-height: 95vh;
```
Comment (superalex) [inline app/styles/modules/common/attachments.scss]: I think a max-width should be added too here

File: app/partials/attachment/attachments.jade [hunk 2]
```diff
@@ -26,3 +26,5 @@ section.attachments
             span.text(data-type="show", translate="ATTACHMENT.SHOW_DEPRECATED")
             span.text.hidden(data-type="hide", translate="ATTACHMENT.HIDE_DEPRECATED")
             span.more-attachments-num(translate="ATTACHMENT.COUNT_DEPRECATED", translate-values="{counter: '{{ctrl.deprecatedAttachmentsCount}}'}")
+
+    div.lightbox.lightbox-block(tg-lb-attachment-preview, ng-model="attachment")
```
Comment (superalex) [inline app/partials/attachment/attachments.jade]: Why is this needed?

**Comment by bameda** [comment]
Add you to AUTHORS.rst and update the CHANGELOG.md

File: app/partials/attachment/attachments.jade (hunk 2 above)
Comment (brettp) [inline app/partials/attachment/attachments.jade]: The div needs to be present to load the lightbox directive. If there's a better way to do this, let me know. I'm still new to angular :-)

Comment (superalex) [inline app/partials/attachment/attachments.jade]: Sorry, I wasn't explaining clearly, the div needs to be present and the directive too but why the ng-model="attachment" par is needed?

**Comment by brettp** [comment]
Updated with requested changes.

**Comment by bameda** [comment]
Move to https://github.com/taigaio/taiga-front/pull/626

I like it :dancers:
